test/extended/builds/cluster_config.go

```diff
@@ -101,5 +104,41 @@ var _ = g.Describe("[Feature:Builds][Serial][Slow][Disruptive] alter builds via
 			})
 
 		})
+
+		g.Context("build config context", func() {
+
+			g.It("Apply default proxy configuration to source build pod through env vars", func() {
+				g.By("apply proxy cluster configuration")
+				err := oc.AsAdmin().Run("apply").Args("-f", invalidproxyConfigFixture).Execute()
+				o.Expect(err).NotTo(o.HaveOccurred())
+				g.By("waiting 1s for build controller configuration to propagate")
```

Comment:
Wouldn't it be better to check the configuration change and wait for it to propagate instead of waiting for one second? This doesn't seem very bullet proof.

Reply:
@coreydaley @wewang58 if IIRC the openshift-controller-manager reads the proxy configs live, so updating the build config object won't trigger a rollout. I think increasing the wait from 1s to 10 seconds is reasonable.